prover/src/zk_circuits_handler/darwin_v2.rs

```diff
@@ -99,6 +88,21 @@ impl DarwinV2Handler {
         Ok(handler)
     }
 
+    fn get_degrees(prover_types: &std::collections::HashSet<ProverType>) -> Vec<u32> {
+        fn get_degrees_inner(prover_type: &ProverType) -> Vec<u32> {
+            match prover_type {
+                ProverType::Chunk => prover_darwin::config::ZKEVM_DEGREES.clone(),
```

Comment:
`prover_darwin_v2` here

Reply:
In the `common::get_degrees` you can receive the `get_degrees_inner` method as an argument and supply it from each zk circuits handler.

Reply:
thanks, it makes sense.

Reply:
Changes made, pls have a look again